roseau/load_flow/models/transformers/parameters.py

```diff
@@ -97,15 +98,55 @@ def __eq__(self, other: object) -> bool:
             return (
                 self.id == other.id
                 and self.windings == other.windings
-                and np.isclose(self.sn, other.sn)
-                and np.isclose(self.p0, other.p0)
-                and np.isclose(self.i0, other.i0)
-                and np.isclose(self.uhv, other.uhv)
-                and np.isclose(self.ulv, other.ulv)
-                and np.isclose(self.psc, self.psc)
-                and np.isclose(self.vsc, other.vsc)
+                and np.isclose(self._sn, other._sn)
+                and np.isclose(self._p0, other._p0)
+                and np.isclose(self._i0, other._i0)
+                and np.isclose(self._uhv, other._uhv)
+                and np.isclose(self._ulv, other._ulv)
+                and np.isclose(self._psc, other._psc)
+                and np.isclose(self._vsc, other._vsc)
             )
 
+    @property
+    @ureg.wraps("V", (None,), strict=False)
+    def uhv(self) -> Q_:
+        """Phase-to-phase nominal voltages of the high voltages side (V)"""
+        return self._uhv
+
+    @property
+    @ureg.wraps("V", (None,), strict=False)
+    def ulv(self) -> Q_:
+        """Phase-to-phase nominal voltages of the low voltages side (V)"""
+        return self._ulv
+
+    @property
+    def i0(self) -> float:
+        """The nominal voltages of the transformer (VA)"""
+        return self._i0
+
+    @property
+    @ureg.wraps("VA", (None,), strict=False)
+    def sn(self) -> Q_:
+        """Losses during off-load test (W)"""
+        return self._sn
+
+    @property
+    @ureg.wraps("W", (None,), strict=False)
+    def p0(self) -> Q_:
+        """Current during off-load test (%)"""
+        return self._p0
+
+    @property
+    @ureg.wraps("W", (None,), strict=False)
+    def psc(self) -> Q_:
+        """Losses during short circuit test (W)"""
+        return self._psc
+
+    @property
+    def vsc(self) -> float:
+        """Voltages on LV side during short circuit test (%)"""
+        return self._vsc
+
```

Comment:
The docstring of those methods doesn't mach with the constructor. 
And I think there is a problem in the constructor as well for `sn`, it should be "The nominal power of the transformer"

Reply:
Error in copy-paste. I will correct it.